src/lib.rs

```diff
@@ -59,7 +70,21 @@ pub unsafe extern "C" fn pthread_create(
         return libc::EPERM;
     }
 
-    *native = thread as _;
+    let os_handle = ctru_sys::threadGetHandle(thread);
+    let mut os_thread = MaybeUninit::zeroed();
```

Comment:
You don't need `MaybeUninit` here, you can just initialize to 0.

Reply:
Good point, 🤦